junit-jupiter-api/src/main/kotlin/org/junit/jupiter/api/Assertions.kt

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright 2015-2017 the original author or authors.
+ *
+ * All rights reserved. This program and the accompanying materials are
+ * made available under the terms of the Eclipse Public License v1.0 which
+ * accompanies this distribution and is available at
+ *
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.junit.jupiter.api
+
+import org.junit.jupiter.api.function.Executable
+import org.junit.platform.commons.meta.API
+import org.junit.platform.commons.meta.API.Usage.Experimental
+import java.util.stream.Stream
+
+/**
+ * [Stream] of functions to be executed.
+ */
+internal typealias ExecutableStream = Stream<() -> Unit>
+internal fun ExecutableStream.convert() = map { Executable(it) }
+
+/**
+ * @see Assertions.assertAll
+ * @since 5.0
+ */
+@API(Experimental)
+fun assertAll(executables: ExecutableStream) =
+    Assertions.assertAll(executables.convert())
+
+/**
+ * @see Assertions.assertAll
+ * @since 5.0
+ */
+@API(Experimental)
+fun assertAll(heading : String?, executables: ExecutableStream) =
+    Assertions.assertAll(heading, executables.convert())
+
+/**
+ * @see Assertions.assertAll
+ * @since 5.0
+ */
+@API(Experimental)
+fun assertAll(vararg executables: () -> Unit) =
+    assertAll(executables.toList().stream())
+
+/**
+ * @see Assertions.assertAll
+ * @since 5.0
+ */
+@API(Experimental)
+fun assertAll(heading: String?, vararg executables: () -> Unit) =
+    assertAll(heading, executables.toList().stream())
+
+/**
+ * Example usage:
+ * ```kotlin
+ * val exception = assertThrows<IllegalArgumentException> {
+ *     throw IllegalArgumentException("Talk to a 🦆")
```

Comment:
Okay, thanks @JLLeitschuh. 

It renders correctly as a duck emoji on my copy of Chrome, but unfortunately it still renders as a box (the character an application displays when it cannot identify what the character is supposed to be) in Gmail and on my Android phone.

I wonder if it would be more maintainable to swap the duck emoji out for a non-emoji character or phrase.